app/components/containers/containers.html

```diff
@@ -100,7 +100,7 @@
             </tr>
           </thead>
           <tbody>
-            <tr dir-paginate="container in (state.filteredContainers = ( containers | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
+            <tr ng-click="container.Checked = !container.Checked; selectItem(container);" ng-class="{active: container.Checked}" dir-paginate="container in (state.filteredContainers = ( containers | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
               <td><input type="checkbox" ng-model="container.Checked" ng-change="selectItem(container)"/></td>
```

Comment:
The checkbox is not working anymore. It's working, but the model change is being triggered twice:

* On click on the checkbox, the model is updated
* The `ng-click` in the `<tr>` is triggered and the model is updated again